tests/test_ERC20.py

```diff
@@ -84,6 +88,30 @@ async def test_transfer(erc20_factory):
     assert execution_info.result.totalSupply == previous_supply
 
 
+@pytest.mark.asyncio
+async def test_transfer_emit_event(erc20_factory):
```

Comment:
```suggestion
async def test_transfer_emits_event(erc20_factory):
```
shouldn't it be plural?

Reply:
note this is the case in many tests not just this one!